packages/engine/lib/execution/src/package/mod.rs

```diff
@@ -0,0 +1,268 @@
+//! # The hEngine Package System
```

Comment:
Don't really know where to leave this comment so just adding it here. I'm a little worried about the folder structure. Previously there was some idea of a "Experiment Packages" and "Simulation Packages" and now we seem to just have "package".

Can you help explain? :)

Reply:
Discussed off-GitHub: We will add experiment packages to this crate as well and make the distinction in the follow-up PR.